corehq/toggles.py

```diff
@@ -106,8 +106,12 @@ def __init__(self, slug, label, tag, namespaces=None, help_link=None,
         # updated.  This is only applicable to domain toggles.  It must accept
         # two parameters, `domain_name` and `toggle_is_enabled`
         self.save_fn = save_fn
-        self.always_enabled = always_enabled or set()
-        self.always_disabled = always_disabled or set()
+        # For icds-cas domain, toggles are delcared in localsettings statically
+        #   to avoid cache lookups
+        self.always_enabled = (always_enabled or set() |
+            {'icds-cas'} if self.slug in settings.ICDS_CAS_ALWAYS_ENABLED_TOGGLES else {})
+        self.always_disabled = (always_disabled or set() |
+            {'icds-cas'} if self.slug in settings.ICDS_CAS_ALWAYS_DISABLED_TOGGLES else {})
```

Comment:
nit: I think this should say `... else set()` so it's a `set` and not a `dict`. Not sure if the way it's called it makes a difference, but it could.